asv_bench/benchmarks/io/json.py

```diff
@@ -126,29 +126,84 @@ def time_float_int(self, orient):
     def time_float_int_str(self, orient):
         self.df_int_float_str.to_json(self.fname, orient=orient)
 
-    def time_floats_with_int_idex_lines(self, orient):
+
+class ToJSONLines(BaseIO):
+
+    fname = "__test__.json"
+
+    def setup(self):
+        N = 10 ** 5
+        ncols = 5
+        index = date_range("20000101", periods=N, freq="H")
```

Comment:
can you use params here instead? (you can leave setup like this), but the methods then get the params name